dev/com.ibm.ws.ssl/resources/com/ibm/ws/ssl/resources/ssl.nlsprops

```diff
@@ -633,3 +633,19 @@ ssl.client.handshake.error.CWPKI0824E=CWPKI0824E: SSL HANDSHAKE FAILURE:  Host n
 ssl.client.handshake.error.CWPKI0824E.explanation=An error occurred during the SSL handshake.  Host name verification is enabled and server's identity can not be verified.  The host name used to access the server does not match the server certificate's SubjectDN or Subject Alternative Name information.
 ssl.client.handshake.error.CWPKI0824E.useraction=When host name verification is enable the server's being accessed needs to be setup with a certificate contain the proper information needed to verify the server's identity.
 # -------------------------------------------------------------------------------------------------
+ssl.cacerts.error.CWPKI0825W=CWPKI0825W: The trustDefaultCerts attribute is enabled and the default truststore failed to initialize. Only the configured truststore is used, and the default truststore is not used. The [{0}] SSL configuration is used without the default truststore. The exception that is returned is: [{1}].
+ssl.cacerts.error.CWPKI0825W.explanation=The default truststore was not able to be set up because a problem occurred. It is not used for trust. Only the configured SSL configuration is used.
+ssl.cacerts.error.CWPKI0825W.useraction=Ensure the default truststore file exists and is accessible.
+# -------------------------------------------------------------------------------------------------
+ssl.environment.cert.error.CWPKI0826W=CWPKI0826W: An error occurred while trying to obtain the certificate from the [{0}] environment variable. The certificate is not added to the [{1}] truststore.
+ssl.environment.cert.error.CWPKI0826W.explanation=The certificate that is specified in the environment variable cannot be accessed and is not included in the truststore.
+ssl.environment.cert.error.CWPKI0826W.useraction=Ensure that the environment variable is populated with a base 64-bit certificate.
+# -------------------------------------------------------------------------------------------------
+ssl.using.cacerts.CWPKI0827I=CWPKI0827I: The [{0}] SSL configuration uses the default truststore in addition to the configured truststore.
+ssl.using.cacerts.CWPKI0827I.explanation=The default truststore and the truststore that is configured for the SSL configuration are used for trust.
+ssl.using.cacerts.CWPKI0827I.useraction=No action is required.
+# -------------------------------------------------------------------------------------------------
+ssl.client.handshake.cacerts.error.CWPKI0828E=CWPKI0828E: The trustDefaultCerts attribute is enabled and trust was not able to be established by using the default truststore. The extended error message from the SSL handshake exception is: [{0}].
```

Comment:
**change**
The trustDefaultCerts attribute is enabled and trust was not able to be established by using the default truststore. The extended error message from the SSL handshake exception is: [{0}].

**to**
The trustDefaultCerts attribute is enabled but trust was not established by using the default truststore. The extended error message from the SSL handshake exception is: "{0}".